frontend/app/monitoring/ContributorMetrics.scala

```diff
@@ -0,0 +1,41 @@
+package monitoring
+
+import com.gu.salesforce.Tier
+import com.gu.zuora.soap.models.Commands.PaymentMethod
+import com.amazonaws.services.cloudwatch.model.Dimension
+import com.gu.memsub.subsv2.SubscriptionPlan
+import controllers.Subscription
+import model.{ContributionPlanChoice, PlanChoice}
+
+class ContributorMetrics(val backendEnv: String) extends TouchpointBackendMetrics {
+
+  val service = "Contributor"
+
+  def putAttemptedSignUp: Unit ={
```

Comment:
Minor point, but style here is to put a space between `=` and `{` so:
`def putAttemptedSignUp: Unit = {`